src/csp.rs

```diff
@@ -1,7 +1,101 @@
+//! Sets the `Content-Security-Policy` (CSP) HTTP header to prevent cross-site injections
+//!
+//! [read more](https://helmetjs.github.io/docs/csp/)
+//!
+//! [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy)
+//!
+//!
+//! ## Examples
+//! ```
+//! let mut policy = armor::csp::new();
+//! policy
+//! .default_src(armor::csp::Source::SameOrigin)
+//! .default_src("areweasyncyet.rs")
+//! .script_src(armor::csp::Source::SameOrigin)
+//! .object_src(armor::csp::Source::None)
+//! .base_uri(armor::csp::Source::None)
+//! .upgrade_insecure_requests();
```

Comment:
Can this be indented by 4 spaces? Thanks!

Reply:
Yes! Sorry I missed that.